gui/sitemap.lua

```diff
@@ -353,6 +360,92 @@ function SitemapScreen:onDismiss()
     view = nil
 end
 
+
+-- --------------------------------
+-- SitemapToolbarOverlay
+--
+
+SitemapToolbarOverlay = defclass(SitemapToolbarOverlay, overlay.OverlayWidget)
+SitemapToolbarOverlay.ATTRS{
+    desc='Adds widgets to the erase interface to open the mass removal tool',
+    default_pos={x=35, y=-1},
+    default_enabled=true,
+    viewscreens={
+        'dwarfmode'
+    },
+    frame={w=28, h=9},
+}
+
+function SitemapToolbarOverlay:init()
+    local button_chars = {
+        {218, 196, 196, 191},
+        {179, '-', 'O', 179},
+        {192, 196, 196, 217},
+    }
+
+    self:addviews{
+        widgets.Panel{
+            frame={t=0, l=0, w=26, h=5},
+            frame_style=gui.FRAME_PANEL,
+            frame_background=gui.CLEAR_PEN,
+            frame_inset={l=1, r=1},
+            visible=function() return not not self.subviews.icon:getMousePos() end,
+            subviews={
+                widgets.Label{
+                    text={
+                        'Open the sitemap menu.', NEWLINE,
```

Comment:
"the sitemap menu" might not be descriptive enough. How about "the general search interface"